Fix github url strings (org edx -> openedx)

sarina opened the pull request:
## This PR was autogenerated

This pr replaces the old GitHub organization, github.com/edx, with the new GitHub organization, github.com/openedx.

Tagging @openedx/tcril-engineering for review, but others are welcome to provide review.

Ref: https://github.com/openedx/tcril-engineering/issues/42

feanil commented on .github/workflows/upgrade-python-requirements.yml:
```diff
@@ -39,7 +39,7 @@ jobs:
       - name: setup testeng-ci
         run: |
-          git clone https://github.com/edx/testeng-ci.git
+          git clone https://github.com/openedx/testeng-ci.git
```
This repo did not get moved as a part of the transition and still lives in the `edx` org. We should not update this URL.

sarina replied:
Thanks - I'm going to address this after I set up the automation to correctly update pylintrc

sarina replied:
The correct solution is to use the `.github/workflow-templates/upgrade-python-requirements.yml` file, which I've done now.

sarina commented:
@feanil my script now auto-does edx_lint and I removed the only other occurrence I found of github.com/edx per your review. Could you take a look/merge then I can continue with my script? (it will be easier if this one merges before I continue)

sarina committed:
fix: update path to .github workflows to read from openedx org

sarina committed:
build: use openedx standard upgrade-python-requirements workflow

This workflow is defined in the .github repo, under the
workflow-templates directory. It should not be hand-edited.

sarina committed:
chore: run edx_lint update with the current version of the repo.

aht007 commented:
@sarina I can see that @openedx/arbi-bom was removed from the team-reviewers due to this transition but no other user/team was added for review. This would mean that requirements upgrade PR would go unnoticed. @openedx/2u-aperture can you please put yourself as the reviewing team in the workflow. Thanks

sarina commented:
This is a pull request closed 2 months ago. I am confused why comments are being added.
